pyproject.toml

```diff
@@ -37,6 +44,7 @@ sphinx-icon = "^0.1.2"
 sphinx-favicon = "^0.2"
 sphinxemoji = "^0.2.0"
 sphinxext-opengraph = "^0.6.3"
+reuse = "^1.0.0"
```

Comment:
I notice, that reviewing `poetry.lock` manually is a bit tricky: You only added `reuse` here, but `poetry.lock` also naturally also contains the direct and indirect dependencies ... which I could only check by resolving that tree on my side, e.g. by running poetry. At least there are only additions of new packages, no changes to previously present ones, so it looks clean from that angle.

Reply:
Yes, I agree that this is the tricky bit. Would something like a compressed BOM be easier for review? I'd assume that this could be generated from poetry.lock automatically.